packages/psss/src/containers/Tables/CountryReportTable.js

```diff
@@ -26,17 +34,45 @@ const VerifiableBody = props => {
   return <TableBody TableRow={Row} {...props} />;
 };
 
-const LayoutRow = styled.div`
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-  padding: 1rem 0;
+const FormRow = styled(FlexStart)`
+  flex: 1;
+  padding-left: 1rem;
 `;
 
 const GreyHeader = styled(FakeHeader)`
   border: none;
 `;
 
+const Alert = styled(SmallAlert)`
+  margin-bottom: 1.2rem;
+`;
+
+const GreyAlert = styled(SmallAlert)`
```

Comment:
I have just styled the grey alert as a one off as I think it is too specific for ui-components.